proctor-common/src/main/java/com/indeed/proctor/common/ProctorUtils.java

```diff
@@ -596,7 +595,7 @@ private static ConsumableTestDefinition defaultFor(final String testName, @Nonnu
     }
 
     public static ProvidedContext convertContextToTestableMap(Map<String,String> providedContext) {
-        final ExpressionFactory expressionFactory = new ExpressionFactoryImpl();
+        final ExpressionFactory expressionFactory = ExpressionFactory.newInstance();
```

Comment:
By using `ExpressionFactory.newInstance`, this code will work with an alternative Java EL implementation (i.e. `org.glassfish.web:javax.el`).